### PR TITLE
Remove redundant initial of hasOwnProperty

### PR DESCRIPTION
**Pull request by behnammodi**
- All test passed

### CODE REVIEW

**Comment by sebmarkbage** [comment]
Yea I've thought of doing this for all the ones we add this way. isArray is another one. There are more added since this PR too.

**Comment by behnammodi** [comment]
@sebmarkbage Do you mean I can create new PR for `isArray` too?

**Comment by sebmarkbage** [comment]
Yea or add it to this one. And if you find any other similar ones, those would be appreciated too.

**Comment by behnammodi** [comment]
All right I will do it

File: packages/react-dom/src/__tests__/ReactCompositeComponent-test.js [hunk 1]
```diff
@@ -6,9 +6,10 @@
  *
  * @emails react-core
  */
-
 'use strict';
 
+import hasOwnProperty from 'shared/hasOwnProperty'
```
Comment (gaearon) [inline packages/react-dom/src/__tests__/ReactCompositeComponent-test.js:11]: Don't do this in tests. This prevents us from being able to test compiled bundles, or at least introduces difficulties.

**Comment by sebmarkbage** [comment]
I think you need to pull and rebase your changes since there has been new callers added since you last opened the PR: https://github.com/facebook/react/blob/master/packages/react-dom/src/server/ReactDOMServerFormatConfig.js#L49

File: packages/react-dom/src/__tests__/ReactCompositeComponent-test.js (hunk 1 above)
Comment (behnammodi) [reply packages/react-dom/src/__tests__/ReactCompositeComponent-test.js:11]: Ok i will revert this

**Comment by behnammodi** [comment]
> I think you need to pull and rebase your changes since there has been new callers added since you last opened the PR: https://github.com/facebook/react/blob/master/packages/react-dom/src/server/ReactDOMServerFormatConfig.js#L49

I did it

**Comment by sebmarkbage** [comment]
Thanks!

**Comment by behnammodi** [comment]
@sebmarkbage You're welcome so for isArray I create a new PR ok?

**Comment by gaearon** [comment]
Sure

**Comment by behnammodi** [comment]
@gaearon Ok

**Pull request merged**